pyro/distributions/lowrank_mvn.py

```diff
@@ -49,6 +49,7 @@ class LowRankMultivariateNormal(TorchDistribution):
     has_rsample = True
 
     def __init__(self, loc, W_term, D_term, trace_term=None):
+        W_term = W_term.t()
```

Comment:
This is a simple fix. Otherwise, we have to change the computation of log_prob, covariance_matrix,... which is unnecessary because we will delete this implementation at the end.

Reply:
Please also update the docstring above and error message below, so as to inform existing users of the breaking change you are making.